benchmarks/src/main/scala/scalaz/zio/ArrayFillBenchmark.scala

```diff
@@ -36,6 +36,25 @@ class ArrayFillBenchmarks {
       } yield ()
     )
   }
+
+  @Benchmark
+  def monoArrayFill() = {
+    import reactor.core.publisher.Mono
+
+    def arrayFill(array: Array[Int])(i: Int): Mono[Unit] =
+      if (i >= array.length) Mono.just(())
```

Comment:
You should use the lazy version of `just`, whatever that is.

Reply:
An approximation would be: `flatten(just(just(()))`, perhaps?

Reply:
.fromCallable(() => ()) ?